opencti-platform/opencti-front/tests_e2e/dashboard/createDashboard.spec.ts

```diff
@@ -8,7 +8,7 @@ test('Create a new dashboard page', async ({ page }) => {
   const dashboardDetailsPage = new DashboardDetailsPage(page);
   const dashboardForm = new DashboardFormPage(page);
   await page.goto('/dashboard/workspaces/dashboards');
-  await dashboardPage.openButtonModal().hover();
+  // await dashboardPage.openButtonModal().hover();
```

Comment:
probably remove the comment here

Reply:
Thanks a lot for your contribution. 
We have completly reworked this tests [dashboard.spec.ts:](https://github.com/OpenCTI-Platform/opencti/blob/master/opencti-platform/opencti-front/tests_e2e/dashboard/dashboard.spec.ts)
Could you remove it?